yklua/LICENSE

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
What file(s) does this license apply to?

Reply:
I thought that if I add this license in yklua directory then it will apply to all the files within it.
What would the convention be for specifying files that this license applies to?

Reply:
> I thought that if I add this license in yklua directory then it will apply to all the files within it.

Yes, but I wasn't sure if it's supposed to apply to those files? If so, it's fine (other than the blank line at the top of the file which we would need to remove :)).